easybuild/easyconfigs/j/json-fortran/json-fortran-8.3.0-GCC-10.3.0.eb

```
@@ -0,0 +1,33 @@
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'json-fortran'
+version = '8.3.0'
+
+homepage = 'https://github.com/jacobwilliams/json-fortran'
+description = "JSON-Fortran: A Modern Fortran JSON API"
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/jacobwilliams/json-fortran/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['5fe9ad709a726416cec986886503e0526419742e288c4e43f63c1c22026d1e8a']
+
+builddependencies = [
+    ('binutils', '2.36.1', '', ('GCCcore', '10.3.0')),
+    ('CMake', '3.20.1', '', ('GCCcore', '10.3.0')),
```

```suggestion
    ('binutils', '2.36.1'),
    ('CMake', '3.20.1'),
```
These dependencies will get picked up automatically (`GCCcore` is a subtoochain of `GCC`)